xline/src/server/xline_server.rs

```diff
@@ -151,6 +147,7 @@ impl XlineServer {
             *self.compact_cfg.compact_batch_size(),
             *self.compact_cfg.compact_sleep_interval(),
             compact_task_rx,
+            self.shutdown_trigger.subscribe(),
         ));
         // TODO: Boot up the compact policy scheduler
```

Comment:
You can remove this comment. We don't need it anymore

Reply:
Removed